src/convertToObject.js

```diff
@@ -6,7 +6,36 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  // Разбиваемо строку
+  const strings = sourceString.split(';');
+
+  strings.forEach((rawStr) => {
```

Comment:
The use of `forEach` is acceptable for iteration, but consider using `reduce` to directly build up the result object instead of modifying an external object. This is a more functional approach and keeps the logic self-contained within the iteration.